packages/core/src/local-workspace/workspace.ts

```diff
@@ -307,7 +307,12 @@ const loadLocalWorkspaceImpl = async ({
       const allEnvSourcesEmpty = envSources.length === 1 && await envSources[0].naclFiles.isEmpty()
       if (allEnvSourcesEmpty) {
         const commonSource = elemSources.sources[elemSources.commonSourceName].naclFiles
-        const { currentEnv } = (await workspaceConfig.getWorkspaceConfig())
+        let { currentEnv } = (await workspaceConfig.getWorkspaceConfig())
+        // Probably workspaceConfig wasn't initialized yet, so we set current env to a default one
+        if (!currentEnv) {
+          currentEnv = ws.currentEnv()
+          await ws.setCurrentEnv(currentEnv)
+        }
```

Comment:
I think it might be enough to just
```suggestion
        const currentEnv = ws.currentEnv()
```
without any other logic, wdyt? (in our flow we are going to set the current env to the newly-created one anyway, and in other flows the user may have set a temporary env just for that command with `-e <env name>` so we shouldn't accidentally change it as a side-effect)

Reply:
Agree, changed and checked